curve-ansible/roles/format_chunkserver/tasks/include/prepare_chunkserver_with_disk_format.yml

```diff
@@ -19,10 +19,10 @@
   shell: sudo ./chunkserver_ctl.sh deploy all --noConfirm
   async: 0
   poll: 3
-  when: not retain_pool
+  when: not retain_pool | bool
 
 - name: format walfilepool
   shell: sudo ./chunkserver_ctl.sh deploy-wal-pool all --noConfirm
   async: 0
   poll: 3
-  when: not retain_pool and not walfilepool_use_chunk_file_pool
\ No newline at end of file
+  when: not retain_pool | bool and not walfilepool_use_chunk_file_pool | bool
```

Comment:
I think it's better to add some brackets to let the statement more clear.

Reply:
> I think it's better to add some brackets to let the statement more clear.

done